frontend/src/components/SensitiveData/components/MaskingAlgorithmsTable.vue

```diff
@@ -150,13 +153,13 @@ const getAlgorithmMaskingType = (
 };
 
 const onRemove = async (index: number) => {
-  const item = algorithmList.value[index];
+  const item = rawAlgorithmList.value[index];
```

Comment:
should we use `algorithmList.value[index]` ?

Reply:
Nice catch. I think I can change to id instead of index to avoid such issues

Reply:
Done